server/command.go

```diff
@@ -126,6 +136,16 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 
 // runStartCommand runs command to start a Zoom meeting.
 func (p *Plugin) runStartCommand(args *model.CommandArgs, user *model.User, topic string) (string, error) {
+	restrict, err := p.isChannelRestrictedForMeetings(args.ChannelId)
+	if err != nil {
+		p.client.Log.Error("Unable to check channel preference", "ChannelID", args.ChannelId, "Error", err.Error())
+		return "Error occurred while starting meeting", nil
+	}
+
+	if restrict {
+		return "Creating zoom meeting is disabled for this channel.", nil
```

Comment:
When "zoom" is used as a product name it should be capitalized.  This needs to be fixed for all cases.